expr/func_invert.go

```diff
@@ -0,0 +1,51 @@
+package expr
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/schema"
+)
+
+type FuncInvert struct {
+	in GraphiteFunc
+}
+
+func NewInvert() GraphiteFunc {
+	return &FuncInvert{}
+}
+
+func (s *FuncInvert) Signature() ([]Arg, []Arg) {
+	return []Arg{
+		ArgSeriesList{val: &s.in}}, []Arg{ArgSeriesList{}}
+}
+
+func (s *FuncInvert) Context(context Context) Context {
+	return context
+}
+
+func (s *FuncInvert) Exec(dataMap DataMap) ([]models.Series, error) {
+	series, err := s.in.Exec(dataMap)
+	if err != nil {
+		return nil, err
+	}
+
+	for i, serie := range series {
+		series[i].Target = fmt.Sprintf("invert(%s)", serie.Target)
```

Comment:
We can't modify these in place. as it changes the input. Can you add a test that verifies the input is unmodified after running through this function?